Makefile

```diff
@@ -556,7 +556,7 @@ KBUILD_RUSTFLAGS := $(rust_common_flags) \
 		    -Cpanic=abort -Cembed-bitcode=n -Clto=n \
 		    -Cforce-unwind-tables=n -Ccodegen-units=1 \
 		    -Csymbol-mangling-version=v0 \
-		    -Crelocation-model=static \
+		    -Crelocation-model=static -Zfunction-sections=n \
```

Comment:
Is there a kernel config that enables it? If not maybe add one? It really helps with reducing code size for rust.